fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -199,6 +200,7 @@ public void postConstruct() {
         TURTLE_X, TEXT_HTML_WITH_CHARSET })
     public Response head() throws UnsupportedAlgorithmException {
         LOGGER.info("HEAD for: {}", externalPath);
+        hasRestrictedPath(externalPath);
```

Comment:
We probably only need to add this check on `PUT` and `POST`.

Reply:
@awoods I can do that, but we should remember to check later and see if you could access.
```
http://localhost:8080/fcrepo/rest/someResource/fedora:versions
```
and get/head the TimeMap. Not that it would be horrible, but you would probably not get the appropriate headers.

Reply:
I'm thinking we should also cover `DELETE`.